README.md

```diff
@@ -12,6 +12,17 @@ The HashiCorp Terraform Visual Studio Code (VS Code) extension adds syntax highl
 - Closes braces and quotes
 - Includes `for_each` and `variable` syntax shortcuts (`fore`, `vare`, `varm`)
 
+## Getting Started
+
+**IMPORTANT:** After installing, you must perform a `terraform init` to provide `terraform-ls` with an up-to-date Provider schema. The language-server will not work correctly without first completing this step!
+
+1. Install the extension from the Marketplace
+1. Reload VS Code after the installation (click the reload button next to the extension).
```

Comment:
Maybe too much context - unsure - but we could link to https://github.com/hashicorp/terraform-ls/issues/222 here - i.e. this step should not be necessary anymore once that issue is resolved

Reply:
I'd say we can leave out for these instructions, and just remove it from the steps once we enhance/add monitoring